base/db/R/insert.format.vars.R

```diff
@@ -0,0 +1,28 @@
+#' Insert Format and Format-Variable Records 
+#' @param con Bety connection object
+#' @param formats_variables_df A 'data.frame' consisting of entries that correspond to columns in the formats-variables table
+#' @param formats_df A 'data.frame' consisting of entries that correspond to columns in the formats table
```

Comment:
1. I guess I didn't envision that the formats entries needed to be in a data frame and, given that the number of entries is modest, I tend to think it's better to pass things in individually here so that it's more clear what the inputs actually are. As noted by someone previously, you should also make explicit that the format args need to be single values, not vectors (i.e. insert.format.vars can't create multiple new format records at once)
2. For anything that _is_ passed as a dataframe, you need to very explicitly document the names of variables, their units/formats, what variables are required, and what variables are optional. You can't expect the user of this function to go login to psql and look at the raw schema to figure out how to format things.
3. There should also be basic error checking on that dataframe so that you don't go and insert something that's going to be invalid. Might also be good to have some error handling in case the inserts themselves throw errors

Reply:
Good point. I'll change the tables to just variable entries. Do you think that I should pass the formats-variables as a data.frame? It too only has a handful of inputs. Just wondering for consistency's sake. Thanks!!

Reply:
As mentioned previously, I think formats_variables should be a data.frame specifically because you are adding multiple entries and that information needs to be organized in a table -- there's too much room for error when you pass separate vectors